var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -86,6 +86,7 @@ class Openmpi(AutotoolsPackage):
     version('3.1.0', '0895e268ca27735d7654bf64cee6c256')  # libmpi.so.40.10.0
 
     # Still supported
+    version('3.0.2', '098fa89646f5b4438d9d8534bc960cd6')  #
```

Comment:
Can you add a comment for which library version gets installed?

Reply:
Will do.